src/cli.rs

```diff
@@ -0,0 +1,11 @@
+use clap::Parser;
+
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about=None)]
+pub(crate) struct Cli {
+    #[arg(short, long="force-choose", help="Force runfast to choose a new runner, instead of \
+        looking for one that may already be set")]
+    pub(crate) force_choose_new: bool,
+    #[arg(short, long="config-path", help="Load specific toml config", default_value="~/.config/runfast/runners.toml")]
```

Comment:
i would prefer this to be called `runners-path` instead of `config-path` since I can envision having a separate config file in the future.